package.json

```diff
@@ -20,13 +20,16 @@
   "private": false,
   "license": "MIT",
   "dependencies": {
-    "prop-types": ">=15.0",
+    "prop-types": ">=15.0"
+  },
+  "peerDependencies": {
     "react": ">=0.14"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
     "babel-preset-env": "^1.6.0",
-    "babel-preset-react": "^6.24.1"
+    "babel-preset-react": "^6.24.1",
+    "react": ">=0.14"
```

Comment:
React doesn't need to be here devDependencies. The example app itself depends on React, and uses webpack to build it (and assumes that react-map-interaction is npm linked). And when in development of the actual source of this package we don't need React since we only run `babel` on the source index.jsx, which doesn't try to do any bundling.